recipes-bsp/u-boot/u-boot_2019.04.bbappend

```diff
@@ -24,16 +24,14 @@ SRC_URI += "file://0001-riscv-add-infrastructure-for-calling-functions-on-ot.pat
 
 DEPENDS += "u-boot-tools-native"
 
-UBOOT_ENTRYPOINT = "0x80200000"
-
 # Overwrite this for your server
 TFTP_SERVER_IP ?= "127.0.0.1"
 
 do_configure_prepend() {
     sed -i -e 's,@SERVERIP@,${TFTP_SERVER_IP},g' ${S}/include/configs/sifive-fu540.h
 
     if [ -f "${WORKDIR}/${UBOOT_ENV}.txt" ]; then
-        mkimage -A arm -O linux -T script -C none -n "U-Boot boot script" \
+        mkimage -O linux -T script -C none -n "U-Boot boot script" \
```

Comment:
you did not describe this change ?

Reply:
@kraj  It seem "-A arm" option removed.  plz anyone tell me what's that option work in line?

Reply:
It's just a small fix to remove the incorrect and unneeded arch when running mkimage on the script.

Reply:
I have pushed a v2 that updates the commit message. Let me know if you think it should be in a separate patch.